index.js

```diff
@@ -23,6 +23,14 @@ function watch(glob, options, cb) {
 
   var opt = assignWith({}, defaults, options, assignNullish);
 
+  if (!opt.events) {
+    opt.events = ['add', 'change', 'unlink'];
+  }
+
+  if (!(opt.events instanceof Array)) {
```

Comment:
use `Array.isArray()` because it's more reliable and available in all supported platforms.